libobs-d3d11/d3d11-subsystem.cpp

```diff
@@ -2826,6 +2826,23 @@ extern "C" EXPORT gs_texture_t *device_texture_open_shared(gs_device_t *device,
 	return texture;
 }
 
+extern "C" EXPORT gs_texture_t *device_texture_open_shared1(gs_device_t *device,
```

Comment:
Not a very explanatory function name, maybe something like `device_texture_open_nt_shared` would be better?